packages/eslint-plugin-js/rules/comma-spacing/ts-comma-spacing.test.ts

```diff
@@ -410,19 +410,6 @@ run({
         },
       ],
     },
-    {
```

Comment:
Conflicts with valid test case in the js version
https://github.com/eslint-stylistic/eslint-stylistic/blob/240006cd887de265dc33eb027765846bd877bfd2/packages/eslint-plugin-js/rules/comma-spacing/comma-spacing.test.ts#L89